DOCS-7624-applied template (#218)
mulesoft/docs-api-manager: modules/ROOT/pages/apply-configure-xml-threat-task.adoc

>s|Policy Name|XML Threat Protection
>s|Summary|Protects against malicious XML in API requests
>s|Category|Security

> >s|Available from Mule version|3.8.0

Reviewer: "First Mule version available"?

Author: We have been doing "Available from Mule version" in other policies. Should we stick to that?

>s|Returned Status Codes|400 - Bad Request
|===

> Applications processing XML requests are susceptible to attacks characterized by unusual inflation of elements, attributes, and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic swings in the size of the application data often signal a security problem. The XML threat protection policy helps protect your applications from such intrusions.

Reviewer: ```suggestion
Applications processing XML requests are susceptible to attacks characterized by unusual inflation of elements, attributes, and nesting levels. Attackers use recursive techniques to consume memory resources. Dramatic increases in the size of the application data often signal a security problem. The XML threat protection policy helps protect your applications from such intrusions.
```
Is "the XML threat protection policy" the same as "the JSON Threat Protection policy"?
If so, try to stick with just one name.
If not...be sure that the first is a common noun ("a XML threat protection policy") or a proper noun, like the JSON version.

Author: Done! Thanks!

If you find that attacks on your Anypoint Platform setup are difficult to detect, design your services architecture with layers of protection in addition to JSON Threat Protection.

== Configuring Policy Parameters

When you apply the XML Threat Protection policy to your API from the UI, the following parameters are displayed:

[width="100%", cols="2,6,1,1"]
|===
s|Field s|Description s|Default s|Required

> |Maximum Node Depth|Specifies the maximum node depth of an XML document ^|0 ^|false

Reviewer: period
```suggestion
|Maximum Node Depth|Specifies the maximum node depth of an XML document. ^|0 ^|false
```

Author: Done! Thanks!

> |Maximum Attribute Count Per Element|Specifies the maximum number of attributes in an element. Note that attributes used for defining namespaces are not counted ^|0 ^|false

Reviewer: ditto
```suggestion
|Maximum Attribute Count Per Element|Specifies the maximum number of attributes in an element. Note that attributes used for defining namespaces are not counted. ^|0 ^|false
```

Reviewer: Please supply a period at the end of each of the following description texts, as in the previous two edits.

Author: Done! Thanks!

|Maximum Child Count|Specifies the maximum number of children of an element in the XML document ^|0 ^|false
|Maximum Text Length|Specifies the maximum length (in characters) of text nodes in the XML document ^|0 ^|false
|Maximum Attribute Length|Specifies the maximum length (in characters) of an attribute in the XML document ^|0 ^|false
|Maximum Comment Length|Specifies the maximum number of comment characters in the XML document ^|0 ^|false
|===

> NOTE: Configuring a value of -1 means unlimited.

Reviewer: ```suggestion
NOTE: A value of -1 indicates that the field value has no limits.
```

Author: Done! Thanks!

== Example

> The following screenshot shows an example of the parameters configured for the XML Threat Protection policy:

Reviewer: Ah! The policy name is init. capped here, so likely should be above.

Author: Yes changed that! Done! Thanks!
